python/graphstorm/model/utils.py

```diff
@@ -582,9 +582,14 @@ def load_sparse_emb(num_embs, ntype_path):
                         "malicious pickle data which will execute arbitrary code " \
                         "during unpickling. Only load data you trust or " \
                         "update torch to 1.13.0+")
-                load_sparse_emb(num_embs, os.path.join(model_path, ntype))
-            else:
-                load_sparse_emb(num_embs, os.path.join(model_path, ntype))
+            emb_path = os.path.join(model_path, ntype)
+            # If we are loading the model from BERT fine-tuning, we may not have
+            # the sparse embeddings.
+            if os.path.exists(emb_path):
```

Comment:
If we are loading model from BERT fine-tuning, why do we need to load the sparse embeddings?

Reply:
when we fine-tune the BERT model with link prediction on the node types with text features, no sparse embeddings are saved in the fine-tuning step. When we need to train the GNN model with fine-tuned BERT model, we need to load the parameters in the input encoder. At this stage, the input encoder can have learnable embeddings. But there are no sparse embeddings saved from the previous step.